src/covid_moh_mali/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -2,4 +2,4 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@mipmap/ic_launcher_background"/>
     <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
-</adaptive-icon>
\ No newline at end of file
+</adaptive-icon>
```

Comment:
So checkstyle throws a warning/info message referring that these files don't have a new line at the end of the file.